Classes/Issues/Comments/Reactions/IssueCommentReactionCell.swift

```diff
@@ -156,6 +156,23 @@ UICollectionViewDelegateFlowLayout {
         let model = reactions[indexPath.item]
         cell.label.text = "\(model.content.emoji) \(model.count)"
         cell.contentView.backgroundColor = model.viewerDidReact ? Styles.Colors.Blue.light.color : .clear
+        
+        var users = model.users
+        guard users.count > 0 else { return cell }
+        
+        let difference = model.count - users.count
+        if difference > 0 {
+            let plural = difference > 1 ? "s" : ""
+            users.append("\(difference) other\(plural)")
```

Comment:
Check out how I'm doing localized plural with date formats. We should do the same here.